resources/lang/en/wiki.php

```diff
@@ -31,5 +31,10 @@
             'link' => 'Show on GitHub',
             'refresh' => 'Refresh',
         ],
+
+        'outdated' => [
+            '_' => 'This page is outdated. Check :default for latest update and perhaps help translating it!',
```

Comment:
`This page contains an outdated translation of the original content. Please check the :default for the most accurate information (and consider updating the translation if you are able to help out)!`